package.json

```diff
@@ -29,5 +29,9 @@
   },
   "workspaces": [
     "apps/*"
-  ]
+  ],
```

Comment:
You can remove these dependencies. They are exclusively used in the backend, and are already listed in the BeAlcoholic/apps/backend/package.json file.

Reply:
okay thanks, i resolved it